addons/project/views/project_views.xml

```diff
@@ -1366,19 +1357,25 @@
                                     </div>
                                 </div>
                                 <div class="o_kanban_record_bottom" t-if="!selection_mode">
-                                    <div class="oe_kanban_bottom_left">
-                                        <field name="priority" widget="priority"/>
-                                        <field name="activity_ids" widget="kanban_activity"/>
+                                    <div class="oe_kanban_bottom_left" t-attf-class="{{record.is_closed.raw_value ? 'opacity-50' : ''}}">
+                                        <field name="priority" widget="priority" style="margin-right: 5px;"/>
+                                        <field name="activity_ids" widget="kanban_activity" style="padding-top: 1.5px;"/>
                                         <b t-if="record.rating_active.raw_value and record.rating_count.raw_value &gt; 0" groups="project.group_project_rating">
-                                            <span style="font-weight:bold;" class="fa fa-fw mt4 fa-smile-o text-success" t-if="record.rating_avg.raw_value &gt;= 3.66" title="Average Rating: Satisfied" role="img" aria-label="Happy face"/>
-                                            <span style="font-weight:bold;" class="fa fa-fw mt4 fa-meh-o text-warning" t-elif="record.rating_avg.raw_value &gt;= 2.33" title="Average Rating: Okay" role="img" aria-label="Neutral face"/>
-                                            <span style="font-weight:bold;" class="fa fa-fw mt4 fa-frown-o text-danger" t-else="" title="Average Rating: Dissatisfied" role="img" aria-label="Sad face"/>
+                                            <span style="font-weight:bold; padding-top:1.5px; padding-right:3px; font-size:16px;" class="fa fa-fw fa-smile-o text-success" t-if="record.rating_avg.raw_value &gt;= 3.66" title="Average Rating: Satisfied" role="img" aria-label="Happy face"/>
```

Comment:
Such a pity that we have to write that much style. Anyway, there is no other way.
However, we could create a class `rating_face` or something, and use a style sheet to set the the style on all 3 at once.

Reply:
I will work on this